<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>traindlg</class>
 <widget class="QDialog" name="traindlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Training</string>
  </property>
  <widget class="QLabel" name="progresslabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>175</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BeginButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>175</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
    </font>
   </property>
   <property name="text">
    <string>Begin!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
